### DISCUSSION TITLE
Local installation demangler not working

### DISCUSSION THREAD

**Discussion by luni64** [compiler-explorer/compiler-explorer#2946] [Q&A]
I did a local install on Win10 following your instructions. It basically works, but I can't get the demangler working . Is there any documentation how to set it up correctly? Did I miss something?

![image](https://user-images.githubusercontent.com/12611497/133920156-5a1bf026-753d-4b79-a9c1-775116d3d7fe.png)

I'm only using gcc-arm-none-eabi,  here my c++.local.properties
```
compilers=&gccarm
defaultCompiler=arm541
objdumper=c:/toolchain/gcc/gcc-arm-none-eabi-5_4-2016q3-20160926-win32/bin/arm-none-eabi-objdump.exe
demangler=c:/toolchain/gcc/gcc-arm-none-eabi-5_4-2016q3-20160926-win32/bin/arm-none-eabi-c++filt.exe
needsMulti=false

###############################
# GCC for ARM
group.gccarm.compilers=&gcc32arm
group.gccarm.includeFlag=-I

# 32 bit
group.gcc32arm.groupName=Arm 32-bit GCC
group.gcc32arm.compilers=arm541:arm1020
group.gcc32arm.isSemVer=true

group.gcc32arm.instructionSet=arm32

compiler.arm1020.exe=c:/toolchain/gcc/gcc-arm-none-eabi-10-2020-q4-major/bin/arm-none-eabi-gcc.exe
compiler.arm1020.name=ARM gcc 10.2 (none)
compiler.arm1020.semver=10.2
compiler.arm1020.supportsBinary=false

compiler.arm541.name=ARM gcc 5.4 (none)
compiler.arm541.exe =c:/toolchain/gcc/gcc-arm-none-eabi-5_4-2016q3-20160926-win32/bin/arm-none-eabi-gcc.exe
compiler.arm541.semver=5.4
compiler.arm541.supportsBinary=false

tools=
```
Thanks for any support

**Comment by AbrilRBS** [comment]
Hi! Just to make sure, did you check that the "Demangle identifiers" output filter is toggled ON?

In the compiler pane:
![imagen](https://user-images.githubusercontent.com/5364255/133939815-041a2134-09ca-49a3-a680-f5431390faee.png)

**Comment by luni64** [comment]
Yes, that was activated.

![image](https://user-images.githubusercontent.com/12611497/133960308-abeee0b9-31ed-4300-ac75-583bbee2334d.png)

**Comment by luni64** [comment]
Any news on that?

**Comment by partouf** [comment]
Can you try to set `demanglerType=cpp` in your c++.local.properties file?

**Reply by partouf** [reply]
Can confirm this works with this compiler and these settings. If the `demanglerType` is not set in the local properties, the `c++.win32.properties` will take precedence and CE will assume your demangler is part of MSVC (`undname.exe`).
